doc/source/examples/incompressible-flow/3d_nitsche_mixer_with_pbt_impeller/nitscher_mixer_with_pbt_impeller.rst

```diff
@@ -0,0 +1,279 @@
+      set kinematic viscosity            = 0.001
+    end
+  end
+
+The kinematic viscosity is set to :math:`0.001`. This, combined with the angular velocity of 1 RPS and the diameter of the impeller of 0.3, will allow us to simulate the vessel at :math:`Re=300`.
+
+
+Mesh
+~~~~~
+
+The ``mesh`` subsection specifies the computational grid:
+
+.. code-block:: text
+
+  subsection mesh
+      set type                 = dealii
+      set grid type            = subdivided_cylinder
+      set grid arguments       = 2:0.5:0.5
+      set initial refinement   = 3
+  end
+
+Because we use an immersed boundary approach, we only need to mesh the vessel without considering the impeller. This greatly simplifies the meshing process. The mesh can be generated straightforwardly using the deal.II mesh generator for a subdivided cylinder. We set the half-height and the radius of the vessel to be :math:`0.5`. This will generate a vessel in which the axis of the impeller will be along the :math:`x` coordinate. The bottom of the vessel will be on the :math:`x=-0.5` plane. This is not really intuitive, but this is a consequence of some of the inherent limitations of the mesh generators.
+
+
+Boundary conditions
+~~~~~~~~~~~~~~~~~~~
+
+The ``boundary conditions`` subsection establishes the constraints on different parts of the domain:
+
+.. code-block:: text
+
+    subsection boundary conditions
+    set number                  = 3
+        subsection bc 0
+            set id                = 0
+            set type              = noslip
+        end
+        subsection bc 1
+            set id                = 1
+            set type              = noslip
+        end
+        subsection bc 2
+            set id                = 2
+            set type              = slip
+        end
+    end
+
+
+Three different boundary conditions are considered. On the lateral walls (``id=0``) and the bottom wall (``id=1``), a no-slip boundary condition applies whereas a slip boundary condition is imposed on the free surface at the top of the vessel (``id=2``).
+
+Nitsche immersed boundary parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection is used to insert the immersed geometry and control its motion.
+
+
+.. code-block:: text
+
+  set number of solids = 1
+	set verbosity 			 = verbose
+
+  subsection nitsche solid 0
+	  subsection mesh
+      		set type                 = gmsh
+      		set file name            = pbt.msh
+          set simplex = true
+	  end
+    subsection solid velocity
+	    set Function expression = 0 ; 1*2*3.1416*z ; -1*2*3.1416*y
+    end
+
+   	set beta 			= 10
+    set enable particles motion		= true
+    set particles sub iterations      = 10
+    set calculate torque on solid = true
+  end
+
+First we specify the number of immersed solid objects. In our case, since we only have the PBT, we set ``number of solids=1``.  Setting ``verbosity = true`` enables us to output in the terminal the post-processing done on the solid object.
```

Comment:
```suggestion
First, we specify the number of immersed solid objects. In our case, since we only have the PBT, we set ``number of solids=1``.  Setting ``verbosity = true`` enables the output of the solid object post-processing in the terminal.
```